NWN.Anvil/src/main/API/Object/NwItem.cs

```diff
@@ -54,6 +54,14 @@ public string UnidentifiedDescription
       set => NWScript.SetDescription(this, value, false.ToInt());
     }
 
+    /// <summary>
+    /// Gets if this item can stack.
+    /// </summary>
+    public bool CanStack
+    {
+      get => int.Parse(NWScript.Get2DAString("baseitems", "Stacking", (int)BaseItemType)) > 1;
```

Comment:
Use the parse extension method:

```suggestion
      get => NWScript.Get2DAString("baseitems", "Stacking", (int)BaseItemType).ParseInt() > 1;
```

I also want to update the functionality of the TwoDimArrayFactory so it is easier to use. But this parameter makes sense for now.

Reply:
Also update the changelog :)